Solved-Problems/Two_Sum/Solution.java

```diff
@@ -0,0 +1,15 @@
+import java.util.*;
```

Comment:
It's a good practice to use only imports that are necessary.

```suggestion
import java.util.HashMap;
import java.util.Map;
```